packages/variables/parseVariables.ts

```diff
@@ -23,6 +23,9 @@ const inlineCodeRegex = /\{\{=(.+?)=\}\}/g
 // {{variable}} and ${{{variable}}}
 const variableRegex = /\{\{([^{}]+)\}\}|(\$)\{\{([^{}]+)\}\}/g
```

Comment:
You basically ignored that regex. We need to extend it instead of replacing it with a simpler one

Reply:
```\{\{([^{}]+)\}\}|(\$)\{\{\{([^{}]+)\}\}\}```
@baptisteArno what do you think of this one?
so for input - 
```
${{xyz}} 
${{{abc}}} 
{{pqr}}
```

it gives following matches - 
```
{{xyz}}
${{{abc}}}
{{pqr}}
```

i wonder if it breaks anything?

Reply:
I fixed the issue already 🙏

https://github.com/baptisteArno/typebot.io/commit/3c07041f2f0a03852138b953a51838b81b077895